docs-source/conf.py

```diff
@@ -0,0 +1,99 @@
+import datetime
+import toml
+import os
+
+project_root = os.path.dirname(os.path.dirname(__file__))
+project_metadata = toml.load(
+    os.path.join(project_root, 'pyproject.toml')
+)['tool']['poetry']
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+author = ", ".join(project_metadata['authors'])
+project = project_metadata['name']
+copyright = project_metadata.get('copyright', datetime.datetime.now().year)
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+
+# The short X.Y version.
+version = project_metadata['version']
+# The full version, including alpha/beta/rc tags.
+release = project_metadata['version']
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This patterns also effect to html_static_path and html_extra_path
+exclude_patterns = []
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = 'sphinx'
+
+# If true, `todo` and `todoList` produce output, else they produce nothing.
+todo_include_todos = False
+
+
+# -- Options for HTML output ----------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+
+html_theme = 'alabaster'
+# html_theme = 'sphinx_material'
```

Comment:
This project uses alabaster?

Reply:
Yes, for now. Sphinx Material (and some other themes) was doing some weird things that I don't want to wrestle with right now.

Reply:
Why not the same theme we are using for the slurm docs? It is not perfect, but definitely better than alabaster.
https://github.com/executablebooks/sphinx-book-theme

Reply:
I've created this task to do that work: https://app.clickup.com/t/1nf3zyr